docker/docker-compose.yml

```diff
@@ -170,3 +171,5 @@ volumes:
   wallet-data:
   db-data:
   tails-storage:
+  issuer-api-dev-bin:
```

Comment:
@esune wasn't sure about these volumes, but added them since they were included in the issuer API and it prevented the build

Reply:
Yup, that is correct - thanks for adding them back in, not sure how I forgot those.